lib/hash-tree.js

```diff
@@ -42,12 +42,17 @@ function getFileInfos(fullPath, _visited) {
       try {
         keys = getFileInfos(path.join(realPath, entry), visited);
       } catch (err) {
+        if (err.code !== 'ENOENT') {
+          throw err;
+        }
+
         if (!keysForTreeWarningPrinted) {
           console.warn('Warning: failed to stat ' + path.join(fullPath, entry));
```

Comment:
Thinking about this warning, I wonder if it is at all ever actionable. I can't think of any actionable steps a reader of this warning can take to resolve them.... unless we can think of something that is actionable, I think we should simply remove it.